mshv-ioctls/src/ioctls/vm.rs

```diff
@@ -12,6 +12,24 @@ use std::os::unix::io::{AsRawFd, FromRawFd, RawFd};
 use vmm_sys_util::errno;
 use vmm_sys_util::ioctl::{ioctl_with_mut_ref, ioctl_with_ref};
 
+/// Structure for injecting interurpt
+/// This struct is passed to request_virtual_interrupt function as an argument
+/// Member variable
+///     interrupt_type: Type of the interrupt
+///     apic_id: Advanced Programmable Interrupt Controller Identification Number
+///     Vector: APIC Vetor (entry of Interrupt vector Table i.e IVT)
```

Comment:
The first `Vector` is capitalized while others are not. Please be consistent. Also the second `Vector` is missing a `c` in it. The third `vector` should be capitalized because it is part of a term `Interrupt Vector Table`.